src/components/App.js

```diff
@@ -13,7 +12,7 @@ export default (props) => {
       <Header />
       <SEO title={title} description={description} />
       <Announcement>
-        tBTC will launch on April 27th, 2020. Read more in <a href="https://www.bloomberg.com/news/articles/2020-04-02/bitcoin-s-ethereum-rivalry-could-be-assuaged-with-tbtc-bridge" target="_blank" rel="noreferrer">Bloomberg</a>.
+        tBTC will launch on April 27th, 2020. Read more in <a href="https://www.bloomberg.com/news/articles/2020-04-02/bitcoin-s-ethereum-rivalry-could-be-assuaged-with-tbtc-bridge" target="_blank" rel="noopener noreferrer">Bloomberg</a>.
```

Comment:
From my reading, `noreferrer` implies `noopener`, btw.

Reply:
I just looked it up, and I agree with you! I only added it to silence the pesky warning from the compiler.